<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>791</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>1051</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="beginInput">
       <property name="text">
        <string>开始输入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Sure">
       <property name="text">
        <string>输入完毕</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ifExtend">
       <property name="text">
        <string>是否需要扩充</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="getFButton">
       <property name="text">
        <string>得到first和follow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="getLRButton">
       <property name="text">
        <string>得到LR(0)图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ifSLR">
       <property name="text">
        <string>是否符合SLR(1)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="getSLR">
       <property name="text">
        <string>得到SLR(1)分析表</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>191</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>831</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>550</y>
      <width>146</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>保存文件</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="showItem">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>191</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="LRTable">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>831</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="SLRTable">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>831</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
